### PR TITLE
py3 - update for google_calendar.py

### PR DESCRIPTION
**Pull request by raphaunix**
Resolving No module named 'apiclient.discovery' error

Tks!

### CODE REVIEW

**Comment by ultrabug** [comment]
Hi @raphaunix thanks for your contribution to py3status!

Would you mind sharing with us your current google-api-python-client version ?

Because I use this module everyday with version `google-api-python-client-1.8.3-` and do not suffer from this.

**Comment by raphaunix** [comment]
Hi @ultrabug, I'm using version 1.11.0
Thanks :)

File: py3status/modules/google_calendar.py [hunk 1]
```diff
@@ -139,7 +139,9 @@
 import os
 import datetime
 
-from apiclient import discovery
+
+#from apiclient import discovery
+from googleapiclient import discovery
```
Comment (ultrabug) [inline py3status/modules/google_calendar.py:144]:
```suggestion
try:
    from googleapiclient import discovery
except ImportError:
    from apiclient import discovery
```

**Comment by ultrabug** [comment]
@raphaunix then it means they changed something I guess :)

I just suggested a code change (please do not comment old code) so that your PR supports newer imports without breaking older ones ;)

**Comment by raphaunix** [comment]
Sorry @ultrabug , it was a stupid test.

Now that you have told me which version you use, a more smooth path may be to create a **requirements.txt**:

`google-api-python-client == 1.8.3`

what do you think?

Thanks

**Comment by ultrabug** [comment]
@raphaunix we decided that py3status modules dependencies would not fall into being required by the project. It is the users' responsibility to install them on their hand. So no requirements.txt please.

**Comment by raphaunix** [comment]
@ultrabug, I get the following error on [**Travis CI**](https://travis-ci.org/github/ultrabug/py3status/jobs/724693256) for py36:
```
error: cannot format py3status/py3.py: INTERNAL ERROR: Black produced code that is not equivalent to the source.  Please report a bug on https://github.com/psf/black/issues.
This diff might be helpful: /tmp/blk_79mllg8d.log
Oh no! 💥 💔 💥
114 files would be reformatted, 22 files would be left unchanged, 1 file would fail to reformat.
ERROR: InvocationError for command /home/travis/build/ultrabug/py3status/.tox/py36/bin/black --diff --check py3status/ (exited with code 123)
___________________________________ summary ____________________________________
ERROR:   py36: commands failed
The command "tox" exited with 1.
```
Should I change anything in the tests?
Thanks

**Comment by ultrabug** [comment]
Hey @raphaunix ; no indeed the new release of black changed the behavior of some formatting and master is affected too!

I shall fix master first, gimme a few

**Comment by ultrabug** [comment]
For you, future reader of this PR: https://github.com/psf/black/issues/1672

**Comment by ultrabug** [comment]
Here we go :) Thanks mate!

**Comment by raphaunix** [comment]
Thanks :)

**Pull request merged**